src/main/java/uk/gov/hmcts/dts/fact/services/admin/AdminCourtAddressService.java

```diff
@@ -376,4 +378,15 @@ private void deleteExistingSecondaryAddressTypes(String slug) {
                                                                        .collect(toList()));
     }
 
+    /**
+    * Validate court address epim ids.
+    * @param courtAddresses The court addresses
+    * @return epim id if invalid else null
+     */
+    public String validateCourtAddressEpimId(List<CourtAddress> courtAddresses) {
+        if (!CollectionUtils.isEmpty(courtAddresses)) {
```

Comment:
court addresses is already checked for being empty in a previous method (validateCourtAddressPostcodes)

Reply:
changed